src/Core/Checkout/Document/Renderer/StornoRenderer.php

```diff
@@ -159,17 +165,36 @@ public function render(array $operations, Context $context, DocumentRendererConf
                     $context,
                     $order->getSalesChannelId(),
                     $order->getLanguageId(),
-                    $locale->getCode()
+                    $locale->getCode(),
                 );
 
                 $doc = new RenderedDocument(
-                    $html,
+                    $html, // @deprecated tag:v6.7.0 - html argument will be removed
                     $number,
                     $config->buildName(),
                     $operation->getFileType(),
                     $config->jsonSerialize(),
                 );
 
+                // set the template renderer to be able to render template
+                $doc->setTemplateOptions([
```

Comment:
Is this needed outside of v6.7.0? If not, then I'd add it inside the if statement.